composer.json

```diff
@@ -54,7 +54,11 @@
     "config": {
         "preferred-install": "dist",
         "sort-packages": true,
-        "optimize-autoloader": true
+        "optimize-autoloader": true,
+        "allow-plugins": {
+            "phpstan/extension-installer": true,
+            "composer/package-versions-deprecated": true
+        }
```

Comment:
This is needed for the Composer 2.2 update. Another time you might consider putting this in a separate commit.